src/components/configuration.rs

```diff
@@ -41,6 +41,7 @@ pub struct Database {
 #[derive(Debug, Deserialize, Clone)]
 pub struct Config {
     pub server: Server,
+    pub rpc_server: Server,
```

Comment:
We could also add these lines for the host and port of the RPC server:
```rust
.set_override_option("rpc_server.host", args.host)?
.set_override_option("rpc_server.port", args.port)?
```
 We already have these for the host and port of the server:
```rust
.set_override_option("server.host", args.host)?
.set_override_option("server.port", args.port)?
```

What do u think?

Reply:
💯